### PR TITLE
Added signers to glossary

### PR DESCRIPTION
**Pull request by Bosch-0**
To get things rolling with https://github.com/BitcoinDesign/Guide/issues/792

### CODE REVIEW

File: guide/glossary/index.md [hunk 1]
```diff
@@ -375,6 +375,12 @@ Since a [private key](#private-key) can be used to prove that the holder control
 One of the most important activities of the bitcoin network is to verify that signatures are valid.
 </div>
 
+### Signer
```
Comment (GBKS) [inline guide/glossary/index.md:378]:
```suggestion
### Signing device
```

Signer is too generic, as discussed in #792. We have to use either signing device or hardware wallet.

**Review by GBKS — CHANGES_REQUESTED** [review]
We cannot use "Signer", it is too generic.

**Review by pavlenex — APPROVED** [review]
LGMT

**Comment by GBKS** [comment]
We have a paragraph about [hardware wallets](https://bitcoin.design/guide/getting-started/hardware/#hardware-wallets) in the "Hardware overview" page, and I'm not clear on how that relates to this glossary entry here. Why not add an "aka signing device" there instead?

**Comment by Bosch-0** [comment]
Lets re-address this when we get to expanding the savings wallet reference design

**Pull request closed without merging**